{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+
+from jinja2 import Template
+import click
+import re
+
+from vivarium_gbd_access.gbd import ARTIFACT_FOLDER
+from vivarium_cluster_tools.psimulate.utilities import get_drmaa
+
+JOB_MEMORY_NEEDED = 50
+JOB_TIME_NEEDED = '24:00:00'
+
+drmaa = get_drmaa()  # safe if not on cluster
+
+
+def create_and_run_job(model_spec_path: Path, output_root: Path):
+    with drmaa.Session() as s:
+        jt = s.createJobTemplate()
+        jt.remoteCommand = "build_artifact"
+        jt.nativeSpecification = '-V -l m_mem_free={}G,fthread=1,h_rt={} -q all.q -P proj_cost_effect'.format(
```

Comment:
Add a todo to template and pick the appropriate project.  We should e.g. be running csu things on proj_csu